include/swift/AST/TypeCheckRequests.h

```diff
@@ -2160,6 +2160,25 @@ class ClosureHasExplicitResultRequest
   bool isCached() const { return true; }
 };
 
+using ProtocolConformanceLookupResult = SmallVector<ProtocolConformance *, 2>;
+void simple_display(llvm::raw_ostream &out, ConformanceLookupKind kind);
+
+/// Lookup and expand all conformances in the given context.
+class LookupAllConformancesInContextRequest :
```

Comment:
How about GetLocalConformancesRequest?

Also can you elaborate in the doc comment on what it means to "expand" them

Reply:
It's not meant to be a general request to look into the conformance tables. This one is specifically about grabbing all of the dependency-relevant conformances so
- TypeCheckProtocol has a way to start checking conformances in the primary
- The evaluator has a cache-point.

Eventually, I would like to have this request call a real `GetLocalConformancesRequest`.

I will update the docs.